### PR TITLE
Fixes bug 1225445 - Added _aggs.product.version parameter to Super Search.

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
@peterbe r?

### CODE REVIEW

File: socorro/external/es/supersearch.py [hunk 1]
```diff
@@ -422,6 +414,29 @@ def get(self, **kwargs):
...
+
+            for param in params['_aggs.product.version']:
+                self._add_second_level_aggs(
+                    param,
+                    lvl2_bucket,
+                    facets_size,
+                    histogram_intervals,
+                )
+
+            search.aggs.bucket(lvl1_field, lvl1_bucket)
```
Comment (peterbe) [inline socorro/external/es/supersearch.py]: I don't understand why you hardcoded this. I can see why you hardcoded the `_aggs.product.version` in the `SearchBase` class but these lines above can all be done with a looping over the keys. 

```python
for key in params:
    if key.startswith('_aggs.'):
        lvl1, lvl2 = key.split('.')[1:]
        ...
```

File: socorro/unittest/external/es/test_supersearch.py [hunk 2]
```diff
@@ -1057,6 +1061,20 @@ def test_get_with_sub_aggregations(self):
         version_sub_facet = res['facets']['signature'][0]['facets']['version']
         eq_(len(version_sub_facet), number_of_crashes)
 
+        # Test with a third level aggregation.
+        kwargs = {
+            '_aggs.product.version': ['_cardinality.signature'],
+        }
+        res = self.api.get(**kwargs)
+
```
Comment (peterbe) [inline socorro/unittest/external/es/test_supersearch.py]: What should happen if you do...
```python
kwargs={'_aggs.xxxxx.yyy': ['_cardinatliy.signature']}
self.api.get(**kwargs)
```

**Comment by peterbe** [comment]
r+ but I think you can refactor that inner part where you break it up. So it's only hardcoded where the key is added to the list of available fields.

File: socorro/unittest/external/es/test_supersearch.py (hunk 2 above)
Comment (AdrianGaudebert) [inline socorro/unittest/external/es/test_supersearch.py]: That would simply be ignored. I was actually wondering, should SuperSearch be more aggressive in validating arguments? At the moment it simply just ignores stuff that it cannot use (like incorrectly spelled field names, or incorrect parameters), and returns some results that might not be what you were expecting. Should it instead raise exceptions when something seems off?

Comment (peterbe) [inline socorro/unittest/external/es/test_supersearch.py]: > Should it instead raise exceptions when something seems off? 

Yes please :)

**Pull request merged**